clickhouse-admin/types/src/lib.rs

```diff
@@ -1200,7 +1200,7 @@ pub enum Timestamp {
 #[derive(Debug, Serialize, Deserialize, JsonSchema, PartialEq)]
 #[serde(rename_all = "snake_case")]
 pub struct SystemTimeSeries {
-    pub time: Timestamp,
+    pub time: String,
```

Comment:
This is seriously doing my head in. Since Timestamp is an _untagged_ enum, serde is having a hard time deserializing. My custom deserializer didn't work, but I'll see if I can find a way